### ISSUE TITLE
Running tests in VS 2015 fails

### ISSUE THREAD

**Issue by hakito** [getgauge/gauge-visualstudio#103]
Installed gauge 0.3.2
Gauge plugin for VS 0.2.1

When running tests in VS 2015 I get the following exception:

```
Test Name:	Vowel counts in multiple word
Test FullName:	[Specification Heading].[Vowel counts in multiple word]
Test Source:	a:\ExtDev\GaugeEvaluation\specs\example.spec : line 1
Test Outcome:	Failed
Test Duration:	0:00:00

Result Message:	
Object reference not set to an instance of an object.
   at Gauge.VisualStudio.TestAdapter.GaugeRunner.BuildOutputPath(String projectRoot)
   at Gauge.VisualStudio.TestAdapter.GaugeRunner.Run(TestCase testCase, Boolean isBeingDebugged, IFrameworkHandle frameworkHandle)
```

Tests run in VS 2013. But with gauge plugin version 0.2.0. (VS 2013 finds no update to 0.2.1)

**Comment by sriv** [comment]
Could you please try Gauge-VisualStudio [v0.2.1](https://visualstudiogallery.msdn.microsoft.com/dd8ea767-aa75-4771-98f4-6c8a9055ca92)? This should be fixed. I am closing this issue, please feel free to reopen if you still face problems.

**Comment by hakito** [comment]
When I reported this issue I already was on Version v0.2.1:

![image](https://cloud.githubusercontent.com/assets/320853/13421445/adffe076-df8e-11e5-8990-6d5c59842bd9.png)

I don't remember how I got v0.2.1 at this time. It would be probably useful If you would provide automatic build numbers in the version, to be sure we are talking about the same version.

Anyway, I can confirm it's working after uninstalling old 0.2.1 and installing new 0.2.1.
